openedx/core/djangoapps/safe_sessions/middleware.py

```diff
@@ -96,9 +96,12 @@
 # .. toggle_name: LOG_REQUEST_USER_CHANGES
 # .. toggle_implementation: SettingToggle
 # .. toggle_default: False
-# .. toggle_description: Turn this toggle on to log anytime the `user` attribute of the request object gets
-#   changed.  This will also log the location where the change is coming from to quickly find issues.
-# .. toggle_warnings: This logging will be very verbose and so should probably not be left on all the time.
+# .. toggle_description: Turn this toggle to track anytime the `user` attribute of the request object gets
+#      changed and store this information on the request. This will also track the location where the change is coming
+#      from to quickly find issues. If user verification fails at response time, all of the information about these
+#      changes will be logged.
+# .. toggle_warnings: If there are many requests failing the user verification check, this logging could get very
+#      verbose very quickly. This toggle is in place as a safety valve to prevent hitting Splunk limits.
```

Comment:
Let me know what you think of the changes. Splunk is edx.org only, so I removed that. Also, the verbose logging was more of a problem when we were logging all changes, not just when we had a verification problem.
```suggestion
# .. toggle_description: Turn this toggle on to provide additional debugging information in the case of a
#      user verification failure. It will track anytime the `user` attribute of the request object gets
#      changed and store this information on the request. This will also track the location where the change is coming
#      from to quickly find issues. If user verification fails at response time, all of the information about these
#      changes will be logged.
# .. toggle_warnings: Adds some processing overhead to all requests to gather debug info. It will also
#      double the logging for failed verification, but this would probably be a smaller concern than the many
#      failed verifications.
```